src/qibo/models/encodings.py

```diff
@@ -92,3 +173,15 @@ def _generate_rbs_pairs(nqubits):
             circuit.add(gates.RBS(*pair, 0.0, trainable=True))
 
     return circuit, pairs_rbs
+
+
+class _probability_distribution_gaussian_loader(rv_continuous):
```

Comment:
Normally I would complain because you are not following the typical class naming rules (which would be `ProbabilityDistributionGaussianLoader`), but I am not sure if it is relevant in this case because it is a very specific scipy usage and `rv_continuous` does not follow it either.

Another possibility to make it private is to move this definition inside the `unary_encoder_random_gaussian` method, but if you are planning to use it in other methods in the future it is better to leave outside.

Reply:
I prefer to leave it outside the function because we will probably define other probability distributions later and it would be not ideal to define all of them inside the function. However, the name style can be changed, true.